src/main/java/seedu/address/logic/commands/find/FindProfileCommand.java

```diff
@@ -0,0 +1,49 @@
+package seedu.address.logic.commands.find;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.commands.util.CommandUtil.getCommandResult;
+import static seedu.address.model.util.ItemUtil.PROFILE_ALIAS;
+import static seedu.address.model.util.ItemUtil.PROFILE_NAME;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.profile.ProfileItemContainsKeywordPredicate;
+import seedu.address.ui.tabs.TabName;
+
+/**
+ * Finds and lists all profileItem in Internhunter whose title contains any of the argument keywords.
+ * Keyword matching is case insensitive.
+ */
+public class FindProfileCommand extends FindCommand {
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " "
+            + PROFILE_ALIAS + ": Finds all profile items whose titles contain any of "
+            + "the specified keywords (case-insensitive).\n"
+            + "Example: " + COMMAND_WORD + " " + PROFILE_ALIAS + " competition html hackathon" + "\n";
+
+    private final ProfileItemContainsKeywordPredicate predicate;
+
+    public FindProfileCommand(ProfileItemContainsKeywordPredicate predicate) {
+        this.predicate = predicate;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
```

Comment:
I think now we can do `model.getFilteredProfileListSize()`. Do you mind helping me and shawn change that portion